File: problemas/2-aumento-salario/python/kauegkluska/aumentosalario.py
print("Digite seu salario")
salario = float(input())


if(salario > 0 and salario <= 400):
    salario = salario + salario*0.15
    porcentagem = "15"
elif(salario > 400 and salario <= 800):
    salario = salario + salario*0.12
    porcentagem = "12"
elif(salario > 800 and salario <= 1200):
    salario = salario + salario*0.10
    porcentagem = "10"
elif(salario > 1200 and salario <= 2000):
    salario = salario + salario*0.07
    porcentagem = "7"
elif(salario > 2000):
    salario = salario + salario*0.04
    porcentagem = "4"

print(f"Você receberá: R$ {salario:.2f} ({porcentagem}%) onde X é o novo salário e Y é o percentual da tabela que foi aplicado no reajuste.")

File: problemas/2-aumento-salario/java/andersonbolduan/AumentoSalario.java
package com.mycompany.salario.reajuste;

import java.util.Scanner;

public class AumentoSalario {

    public static void main(String[] args) {
        
        Scanner entrada = new Scanner(System.in);
        
        System.out.print("Informe o salário atual: ");
        double valorSalario = entrada.nextDouble();
        
        double salarioAtualizado;
        int porcentagem = 0;

        if (valorSalario >= 0 && valorSalario <= 400.00) {
            porcentagem = 15;
        } else if (valorSalario <= 800.00) {
            porcentagem = 12;
        } else if (valorSalario <= 1200.00) {
            porcentagem = 10;
        } else if (valorSalario <= 2000.00) {
            porcentagem = 7;
        } else {
            porcentagem = 4;
        }

        salarioAtualizado = valorSalario + (valorSalario * porcentagem / 100);

        System.out.printf("Você receberá: R$ %.2f (%d%%)%n", salarioAtualizado, porcentagem);
        
        entrada.close();
    }
}
